dynamodb/src/main/scala/akka/stream/alpakka/dynamodb/scaladsl/DynamoDb.scala

```diff
@@ -31,48 +32,29 @@ object DynamoDb {
     Flow[In].mapAsync(parallelism)(operation.execute(_))
 
   /**
-   * Create a Flow that emits a response for every request to DynamoDB.
-   * A successful response is wrapped in [scala.util.Success] and a failed
-   * response is wrapped in [scala.util.Failure].
+   * Create a `FlowWithContext` that emits a response for every request to DynamoDB.
+   * A successful response is wrapped in [[scala.util.Success]] and a failed
+   * response is wrapped in [[scala.util.Failure]].
    *
-   * The returned flow is meant to compose easily with an Akka Stream RetryFlow
-   * which can retry requests, that have responses wrapped in [scala.util.Try].
+   * The context is merely passed through to the emitted element.
    *
    * @param parallelism maximum number of in-flight requests at any given time
-   *
-   * @tparam State the type of the pass-through value that is taken together with
-   *               requests and is emitted together with responses. Can be used,
-   *               for example:
-   *                 * to correlate a request with a response
-   *                 * to track number of retries
-   *                 * to store request to be issued in the case of retry
```

Comment:
I like the explanation here.  It may help motivate people to try `withContext`.  Any reason why you truncated it?

Reply:
These examples are not relevant with the simpler RetryFlow I'm working on.

Reply:
Aren't the examples are applicable for users who use `withContext`, regardless of the `RetryFlow`?

Reply:
Just the first example is, and that should be documented in the `withContext` docs.

Reply:
Ok.  Makes sense.